wordpress-mobile/WordPress-iOS
# Update color palette to 2.5.0

Refs #15852. This PR updates the color palette assets used by the app to the [2.5.0 release of Color Studio](https://github.com/Automattic/color-studio/). There are no code changes, only asset catalog updates. 

The colors were all imported straight from Color Studio's generated xcasset file, with the exception of dark mode variants for blue and wordPressBlue. For these I added a dark appearance alternative for each colour, which I based on the mappings we used previously:

```
- blue0 -> blue0
- blue5 -> blue0
- blue10 -> blue0
- blue20 -> blue5
- blue30 -> blue10
- blue40 -> blue20
- blue50 -> blue30
- blue60 -> blue40
- blue70 -> blue50
- blue80 -> blue60
- blue90 -> blue70
- blue100 -> blue80

- wpblue0 -> wpblue0
- wpblue5 -> wpblue0
- wpblue10 -> wpblue5
- wpblue20 -> wpblue10
- wpblue30 -> wpblue20
- wpblue40 -> wpblue30
- wpblue50 -> wpblue40
- wpblue60 -> wpblue50
- wpblue70 -> wpblue60
- wpblue80 -> wpblue70
- wpblue90 -> wpblue80
- wpblue100 -> wpblue100
```

**To test**

- This will need a design review, but I'll wait until I also have the WordPress deprecation and pink changes included too.
- For this PR, there should mostly just be slight changes to colors, so it's really just a case of looking through the app and ensuring nothing looks super out of place. Check light and dark mode.

cc @mattmiklic 

## Regression Notes

1. Potential unintended areas of impact

- Dark mode colors (but I think I have mapped these over correctly)
- Colors not looking 'right' in various places across the app

2. What I did to test those areas of impact (or what existing automated tests I relied on)

- On the [Color Studio palette page](https://color-studio.blog/) I eyeballed the changes between the version we were using and the new version and I couldn't see anything _drastically_ different.
- I checked through the app in light and dark mode and couldn't see any issues.

3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:

  - Will add release notes with the final set of color changes.
